themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -0,0 +1,64 @@
+@use "uswds-core" as *;
+
+.dg-guide__menu-bar {
+  @include u-border-y("1px", "gray-cool-30", "solid");
+  @include u-z("top");
+  align-items: center;
+  background-color: color("gray-10");
+  display: flex;
+  height: units(10);
+  justify-content: center;
+  position: sticky;
+  top: -1px;
+  white-space: nowrap;
+
+  &-image {
+    margin: 0 units(3);
+
+    &.full {
+      min-width: units("card");
+      width: units("card");
+    }
```

Comment:
Might be misunderstanding, but doesn't the `width` make the `min-width` redundant? Or is it for something else?